src/giskardpy/plugin_update_constraints.py

```diff
@@ -92,9 +98,12 @@ def parse_constraints(self, cmd):
                 c = C(self.god_map, **params)
                 soft_constraints = c.get_constraints()
```

Comment:
Please add an error message, that indicates if the parameter or the .get_constraints were the problem. You probably need separate try catches. Only the 2. one should be an implementation exception. The first one can be whatever the default for wrong parameters in python is.